scripts/test-acceptance.sh

```diff
@@ -53,4 +53,4 @@ if [[ "${test_name}" = "accessibility" ]]; then
 fi
 
 echo "Running acceptance tests from ${test_name}.py against the sandbox..."
-nosetests ../acceptance/${test_name}.py --with-xunit --xunit-file=../acceptance/xunit-${test_name}.xml
+nosetests ../acceptance/${test_name}.py:MultipleOpenAssessmentTest --with-xunit --xunit-file=../acceptance/xunit-${test_name}.xml
```

Comment:
We don't need an accessibility test for every acceptance test. But we should have an accessibility test for having 2 problems on the same page-- it will catch non-unique IDs, among other issues. It would be very helpful to know what issues such a test catches, but we may not be able to submit it until more issues are fixed.

Can you please try adding the accessibility test and let us know what fails?